xstartup.sh

```diff
@@ -8,12 +8,13 @@ echo $PASS | passwd --stdin root
 /usr/sbin/sshd
 
 export DISPLAY=:99
-Xvfb :99 -shmem -screen 0 1280x1024x16 &
-x11vnc -display :99 -N -shared -forever &
 export PATH="/usr/bin:/root/firefox:/root/chrome-driver:$PATH"
 
-# Start the window manager
+Xvfb $DISPLAY -shmem -screen 0 '1280x1024x16' &
+sleep 5
```

Comment:
Yep, x11vnc failed to start locally as Xvfb didn't finish init phase. Not sure if it happens on prod, but it crashed ~50% of local containers